notebooks/tutorials/05_full_tutorial.ipynb

```diff
@@ -129,7 +129,7 @@
       "Requirement already satisfied: PySocks!=1.5.7,>=1.5.6 in /home/gbae/miniconda3/envs/mednist/lib/python3.6/site-packages (from requests[socks]>=2.12.0->gdown) (1.7.1)\n",
       "Downloading...\n",
       "From: https://drive.google.com/uc?id=1GC_N8YQk_mOWN02oOzAU_2YDmNRWk--n\n",
-      "To: /home/gbae/repo/monai-deploy-app-sdk/notebooks/tutorials/ai_spleen_seg_data_updated_1203.zip\n",
```

Comment:
can we replace `/home/<user>` with `~` ?

we should also standardize the path we use. I recommend using `~/src/monai-deploy-app-sdk/`.

Reply:
`~/src/` only works if that directory is created in advance.

We can also use a variable the repopath, eg: `export MONAIREPO=${PWD}`